node/service/src/lib.rs

```diff
@@ -532,7 +532,7 @@ where
 	let config = sc_consensus_grandpa::Config {
 		// FIXME #1578 make this available through chainspec
 		gossip_duration: std::time::Duration::from_millis(333),
-		justification_period: 512,
+		justification_generation_period: GRANDPA_JUSTIFICATION_PERIOD,
```

Comment:
This change is introduced by substrate [PR #14423](https://github.com/paritytech/substrate/pull/14423)